src/main/java/frc/robot/PhotonRunnable.java

```diff
@@ -54,6 +54,8 @@ public class PhotonRunnable implements Runnable {
 
   private final Supplier<Pose2d> poseSupplier;
 
+  public boolean isCameraConnected;
```

Comment:
This field shouldn't be `public`. By making it `public`, its value can be modified outside of the class. You probably should make this `private`, and add an accessor / getter method.